docs/core/tools/dotnet-sln.md

```diff
@@ -131,98 +131,80 @@ dotnet sln [<SOLUTION_FILE>] remove <PROJECT_PATH> [<PROJECT_PATH>...]
-  ```
-
-- Add multiple C# projects to a solution:
-
-  ```dotnetcli
-  dotnet sln todo.sln add todo-app/todo-app.csproj back-end/back-end.csproj
-  ```
-
-- Remove multiple C# projects from a solution:
-
-  ```dotnetcli
-  dotnet sln todo.sln remove todo-app/todo-app.csproj back-end/back-end.csproj
-  ```
-
-- Add multiple C# projects to a solution using a globbing pattern (Unix/Linux only):
-
-  ```dotnetcli
-  dotnet sln todo.sln add **/*.csproj
-  ```
+---
 
-- Add multiple C# projects to a solution using a globbing pattern (Windows PowerShell only):
+### **Examples**
 
-  ```dotnetcli
-  dotnet sln todo.sln add (ls -r **/*.csproj)
-  ```
+#### **Remove a single project from a solution**
+```dotnetcli
+dotnet sln todo.sln remove todo-app/todo-app.csproj
+```
+**Expected Output:**
+```
+Removed project(s) from solution file.
+```
 
-- Remove multiple C# projects from a solution using a globbing pattern (Unix/Linux only):
+#### **Remove multiple projects from a solution**
+```dotnetcli
+dotnet sln todo.sln remove todo-app/todo-app.csproj back-end/back-end.csproj
+```
+**Expected Output:**
+```
+Removed project(s) from solution file.
+```
 
-  ```dotnetcli
-  dotnet sln todo.sln remove **/*.csproj
-  ```
+#### **Remove a project when a solution file is automatically detected**
+```dotnetcli
+dotnet sln remove todo-app/todo-app.csproj
+```
+**Expected Output:**
+```
+Removed project(s) from solution file.
+```
 
-- Remove multiple C# projects from a solution using a globbing pattern (Windows PowerShell only):
+#### **Remove multiple projects using a globbing pattern (Unix/Linux only)**
+```dotnetcli
+dotnet sln todo.sln remove **/*.csproj
+```
 
-  ```dotnetcli
-  dotnet sln todo.sln remove (ls -r **/*.csproj)
-  ```
+#### **Remove multiple projects using a globbing pattern (Windows PowerShell only)**
+```dotnetcli
+dotnet sln todo.sln remove (ls -r **/*.csproj)
+```
 
-- Create a solution, a console app, and two class libraries. Add the projects to the solution, and use the `--solution-folder` option of `dotnet sln` to organize the class libraries into a solution folder.
+---
 
-  ```dotnetcli
-  dotnet new sln -n mysolution
-  dotnet new console -o myapp
-  dotnet new classlib -o mylib1
-  dotnet new classlib -o mylib2
-  dotnet sln mysolution.sln add myapp\myapp.csproj
-  dotnet sln mysolution.sln add mylib1\mylib1.csproj --solution-folder mylibs
-  dotnet sln mysolution.sln add mylib2\mylib2.csproj --solution-folder mylibs
-  ```
+### **Notes**
+- If the solution file **does not exist**, the command will fail with an error.  
+- If a specified project is **not part of the solution**, the command will display a message but continue execution.  
+- **This command does not delete** the project files from disk; it only removes them from the solution file.  
+- If a solution file is not explicitly provided, `dotnet sln remove` will attempt to find one in the current directory.  
```

Comment:
We should add note here to explain the sln vs slnx probing algorithm that I described in the linked issue.